addon/components/u-edit/shared-modules/drag-and-drop.js

```diff
@@ -0,0 +1,88 @@
+import Uploader from '@upfluence/ember-upf-utils/uploader';
+import Configuration from '@upfluence/ember-upf-utils/configuration';
+
+export default class ModuleBuilder {
+  constructor(session, toast) {
+    Object.assign(this, { session, toast });
+  }
+
+  build(editor, element) {
+    return new Module(editor, element, this.session.data.authenticated.access_token, this.toast);
+  }
+}
+
+export class Module {
```

Comment:
So, in the end, the `export` was needed ? 🙂

Reply:
do we really need the export if the only public thing here is the ModuleBuilder ?

Reply:
Mb I forgot to remove that